app/react-native/src/preview/story_store.js

```diff
@@ -21,6 +24,8 @@ export default class StoryStore {
       index: count,
       fn,
     };
+
+    this.emit('storyAdded', kind, name, fn);
```

Comment:
I hadn't spotted the remove method. I can't think of a situation when that would be used...